doc/make_external_gallery.py

```diff
@@ -115,6 +115,10 @@ def format(self):
         description="Stéphane Laurent's blog showing a sample of the animations they realized with PyVista.",
         link="https://laustep.github.io/stlahblog/posts/MyPyVistaArtworks.html",
         image="Duoprism_3-30.gif"),
+    ptera_software=Example(title="PteraSoftware",
+        description="Ptera Software is a fast, easy-to-use, and open-source software package for analyzing flapping-wing flight.",
+        link="https://github.com/camurban/pterasoftware",
+        image=" ptera_software.gif"),
```

Comment:
This might cause a problem
```suggestion
        image="ptera_software.gif"),
```